l10n_it_website_sale_fiscalcode/__init__.py

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Nicola Malcontenti - Agile Business Group
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```

Comment:
Please replace in all headers http://www.gnu.org/licenses/agpl by http://www.gnu.org/licenses/lgpl